src/utils/lodash.ts

```diff
@@ -0,0 +1,4 @@
+import _ from "lodash";
+
+export const mergeCustomizer = (obj: any, src: any): any =>
```

Comment:
Should be a mixin but I was having issues getting mixins to work on TS

Reply:
I already fought with this, check:

https://github.com/EyeSeeTea/vaccination-app/blob/development/src/utils/lodash-mixins.ts

Reply:
Perfect! Thanks :)